### PR TITLE
TELCODOCS-1569: Add rootDeviceHints in the SNO SiteConfig installation reference

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.13, 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1569

Link to docs preview:
https://64884--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites

https://64884--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-manual-install#ztp-generating-install-and-config-crs-manually_ztp-manual-install

https://64884--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites#ztp-deploying-a-site_ztp-deploying-far-edge-sites

QE review:
- [x] QE has approved this change.

Additional information: ACKs are from the original PR, they can be found here: https://github.com/openshift/openshift-docs/pull/62717/files
This update is only needed because of restructuring and removing callouts

### CODE REVIEW

**Comment by Missxiaoguo** [comment]
Thanks!
/lgtm

**Review by JoeAldinger — COMMENTED** [review]
small suggestion

File: modules/ztp-sno-siteconfig-config-reference.adoc [hunk 1]
```diff
@@ -70,6 +70,9 @@ When creating the `bmh-secret` CR, use the same namespace as the `SiteConfig` CR
 |Set the boot mode for the host to `UEFI`.
 The default value is `UEFI`. Use `UEFISecureBoot` to enable secure boot on the host.
 
+|`spec.clusters.nodes.rootDeviceHints`
+|Specifies the device for deployment. Identifiers that are stable across reboots are recommended, for example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. For a detailed list of stable identifiers, see the About root device hints section.
```
Comment (stevsmit) [inline modules/ztp-sno-siteconfig-config-reference.adoc:74]:
```suggestion
|Specifies the device for deployment. Identifiers that are stable across reboots are recommended, for example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. For a detailed list of stable identifiers, see the "About root device hints" section.
```

File: scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc [hunk 2]
```diff
@@ -23,6 +23,11 @@ include::modules/ztp-configuring-kernel-arguments-for-discovery-iso.adoc[levelof
 
 include::modules/ztp-deploying-a-site.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites[Single-node OpenShift SiteConfig CR installation reference]
```
Comment (stevsmit) [inline scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc:29]:
```suggestion
* xref:../../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites[{sno-caps} SiteConfig CR installation reference]
```

File: scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc [hunk 3]
```diff
@@ -29,6 +29,8 @@ include::modules/ztp-generating-install-and-config-crs-manually.adoc[leveloffset
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#root-device-hints_preparing-to-install-with-agent-based-installer[About root device hints]
 
+* xref:../../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites[Single-node OpenShift SiteConfig CR installation reference]
```
Comment (stevsmit) [inline scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc:32]:
```suggestion
* xref:../../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-sno-siteconfig-config-reference_ztp-deploying-far-edge-sites[{sno-caps}  SiteConfig CR installation reference]
```

**Comment by stevsmit** [comment]
@amolnar-rh A few more fits before merger. Thanks!

**Comment by amolnar-rh** [comment]
@stevsmit Updated the PR as per your suggestions. Could you please merge it for main and CP for 4.13 and 4.14 when you have the time?

**Pull request merged**